docs/content/en/lazy-load-translations.md

```diff
@@ -73,3 +73,10 @@ Note that if you want to use the `$axios` instance from the `@nuxtjs/axios` modu
 This rule in fact applies also to any other module that adds plugins and whose functionality you'd want to use from within that function.
 
 </alert>
+
+## `skipNuxtState`
+<badge>v6.3.0+</badge>
+
+In the case of statically generated sites if the option `lazy` is set to `true` the translation messages for the `fallbackLocale` (or `defaultLocale` if no fallback is present) are injected into the Nuxt state. This results in them being included and repeated in the HTML of each generated page. Since each page is unique the same translations are loaded over and over again.
```

Comment:
Note that what is injected into the Nuxt state is the currently selected locale (if different than the fallback locale, since the fallback locale is included in the main bundle).

I'll rewrite it in my PR.